servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -485,8 +568,20 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 		Vector<Color> c;
 		c.push_back(clear_color.to_linear());
 
+		RD::FramebufferFormatID fb_format = RD::get_singleton()->framebuffer_get_format(opaque_framebuffer);
 		RenderListParameters render_list_params(render_list[RENDER_LIST_OPAQUE].elements.ptr(), render_list[RENDER_LIST_OPAQUE].element_info.ptr(), render_list[RENDER_LIST_OPAQUE].elements.size(), reverse_cull, PASS_MODE_COLOR, rp_uniform_set, get_debug_draw_mode() == RS::VIEWPORT_DEBUG_DRAW_WIREFRAME, Vector2(), p_render_data->lod_camera_plane, p_render_data->lod_distance_multiplier, p_render_data->screen_lod_threshold, p_render_data->view_count);
-		_render_list_with_threads(&render_list_params, opaque_framebuffer, keep_color ? RD::INITIAL_ACTION_KEEP : RD::INITIAL_ACTION_CLEAR, will_continue_color ? RD::FINAL_ACTION_CONTINUE : RD::FINAL_ACTION_READ, RD::INITIAL_ACTION_CLEAR, will_continue_depth ? RD::FINAL_ACTION_CONTINUE : RD::FINAL_ACTION_READ, c, 1.0, 0);
+		render_list_params.framebuffer_format = fb_format;
+		if ((uint32_t)render_list_params.element_count > render_list_thread_threshold && false) {
+			// secondary command buffers need more testing at this time
+			//multi threaded
+			thread_draw_lists.resize(RendererThreadPool::singleton->thread_work_pool.get_thread_count());
+			RD::get_singleton()->draw_list_begin_split(opaque_framebuffer, thread_draw_lists.size(), thread_draw_lists.ptr(), keep_color ? RD::INITIAL_ACTION_KEEP : RD::INITIAL_ACTION_CLEAR, will_continue_color ? RD::FINAL_ACTION_CONTINUE : RD::FINAL_ACTION_READ, RD::INITIAL_ACTION_CLEAR, will_continue_depth ? RD::FINAL_ACTION_CONTINUE : RD::FINAL_ACTION_READ, c, 1.0, 0);
```

Comment:
@reduz I copied the setting for the drawlist as they were in the normal pass approach, do they make sense with subpasses?